backend/fms_core/filters.py

```diff
@@ -67,6 +67,7 @@ class Meta:
         }
 
 class SampleFilter(GenericFilter):
+    id__not__in = django_filters.CharFilter(method="id_not_in_filter")
```

Comment:
I tend to think this might be moved to the Generic base class ... We should use it sparingly but it could be used elsewhere ... I tend to think an numeric filter would be faster since this is numeric field in the DB (it possibly has to cast each values to a char)